eng/Analyzers_ShippingRules.ruleset

```diff
@@ -32,4 +32,8 @@
     <!-- TODO: Remove this exclusion and fix related issues, tracked by https://github.com/dotnet/roslyn-analyzers/issues/3915 -->
     <Rule Id="RS0041" Action="None" />
   </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.NetAnalyzers" RuleNamespace="Microsoft.CodeAnalysis.NetAnalyzers">
+    <Rule Id="CA1508" Action="None" />
```

Comment:
Can you please file work items to enable them?

Reply:
Filed #4914 and #4915